src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -264,7 +269,7 @@ public function definitionList(...$list)
     /**
      * {@inheritdoc}
      */
-    public function ask(string $question, ?string $default = null, $validator = null)
+    public function ask(string $question, ?string $default = null, callable $validator = null)
```

Comment:
Adding a typehint on an argument is a BC break.

Reply:
fixed